### PR TITLE
add peter daily profile

### PR DESCRIPTION
**Pull request by thepeted**

### CODE REVIEW

File: people.html [hunk 1]
```diff
@@ -290,6 +290,68 @@ <h6>Tools</h6>
...
+              <div class="col-md-8 twitter-mention-container">
+                <a href="https://twitter.com/intent/tweet?screen_name=peterdaily&ref_src=twsrc%5Etfw" class="twitter-mention-button" data-show-count="false">
+                  Tweet to @peterdaily
+                </a>
+              </div>
+            </div>
+            <div class="row">
+              <div class="col-md-12">
+                <p>
+                  Peter worked in HR and analytical roles for 10 years before discovering the joy of code and changing careers to software develoment.
```
Comment (dangerworm) [inline people.html]: Spelling: development

File: people.html [hunk 2]
```diff
@@ -290,6 +290,68 @@ <h6>Tools</h6>
...
+                  First attending CodeUp in 2015 and working through the <a href="https://freecodecamp.org">FreeCodeCamp</a> curriculum, he secured a
+                  Frontend Developer job at ResponseTap in 2016.  He is a self-confessed JavaScript and React fanboy!
+                </p>
+                <p>Peter helps to run FreeCodeCamp Study Groups across Manchester - including at CodeUp.  Passionate about open source learning,
+                   mountain biking and Nintendo, Peter is also a long suffering Stockport County fan.</p>
+                <p>
+
```
Comment (dangerworm) [inline people.html]: Any reason for the empty paragraph?

Comment (dangerworm) [reply people.html]: I'll sort it. ;)

Comment (thepeted) [reply people.html]: Thanks for catching the typos :-)

**Pull request merged**